src/server/worker/pipeline/transform/worker.go

```diff
@@ -264,16 +264,16 @@ func handleDatumSet(driver driver.Driver, logger logs.TaggedLogger, task *DatumS
 	stats := &datum.Stats{ProcessStats: &pps.ProcessStats{}}
 	if err := pachClient.WithRenewer(func(ctx context.Context, renewer *renew.StringSet) error {
 		// Setup file operation client for output meta commit.
```

Comment:
very minor nit, could we move this comment down to the output meta commit `WithCreateFileSetClient` call.